shared/actions/config/index.js

```diff
@@ -2,14 +2,36 @@
 import * as Constants from '../../constants/config'
 import engine from '../../engine'
 
+import {switchTab} from '../../actions/tabbed-router'
+import {navigateTo} from '../../actions/router'
+import {loginTab, moreTab} from '../../constants/tabs'
+
 // $FlowFixMe
 import * as native from './index.native'
 
 import type {AsyncAction} from '../../constants/types/flux'
 import type {config_getConfig_rpc, config_getExtendedStatus_rpc, config_getCurrentStatus_rpc} from '../../constants/types/flow-types'
 
+function switchTabs(dispatch, getState) {
+  const {config: {status}} = getState()
+  if (!status) {
+    return
```

Comment:
Unclear

On Tuesday, February 9, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In shared/actions/config/index.js
> <https://github.com/keybase/client/pull/1939#discussion_r52391080>:
>
> >  // $FlowFixMe
> >  import * as native from './index.native'
> >
> >  import type {AsyncAction} from '../../constants/types/flux'
> >  import type {config_getConfig_rpc, config_getExtendedStatus_rpc, config_getCurrentStatus_rpc} from '../../constants/types/flow-types'
> >
> > +function switchTabs(dispatch, getState) {
> > +  const {config: {status}} = getState()
> > +  if (!status) {
> > +    return
>
> Is this an error? we should log if so. In case we get a bug report so we
> see it in the log
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1939/files#r52391080>.
>